GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -202,7 +202,9 @@ private bool IsGVFSUpgradeAllowed(out string consoleError)
 
             if (this.IsServiceInstalledAndNotRunning())
             {
-                adviceText = isConfirmed ? $"Run `sc start GVFS.Service` and run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run `sc start GVFS.Service` and run {GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm} from an elevated command prompt.";
+                adviceText = isConfirmed

```

Comment:
Instead of building the sentence from multiple fragments, can we simplify this by just instructing the user to start the service and run the command again? We have already checked they are running from an elevated command prompt.

`To start the service, run {command} and run the command again`